tests/src/test/scala/org/http4s/UriSpec.scala

```diff
@@ -1349,6 +1349,59 @@ class UriSpec extends Http4sSuite {
       assertEquals(pre.concat(post), path1)
     }
 
+    test("empty concat 1")(assertEquals(path"".concat(path""), path""))
+    test("empty concat 2")(assertEquals(path"/".concat(path""), path"/"))
+    test("empty concat 3")(assertEquals(path"".concat(path"/"), path"/"))
+    test("empty concat 4")(assertEquals(path"/".concat(path"/"), path"/"))
+
+    test("simple concat 1")(assertEquals(path"a/".concat(path""), path"a/"))
+    test("simple concat 2")(assertEquals(path"a".concat(path"/"), path"a/"))
+    test("simple concat 3")(assertEquals(path"".concat(path"/a"), path"/a"))
+    test("simple concat 4")(assertEquals(path"/".concat(path"a"), path"/a"))
+
+    test("When left side of concat is absolute then the result is absolute") {
```

Comment:
Not sure if it makes a difference?
https://scalameta.org/munit/docs/integrations/scalacheck.html

```suggestion
    property("When left side of concat is absolute then the result is absolute") {
```

Reply:
Thanks. I didn't know about the `property`.